build.gradle

```diff
@@ -21,7 +21,7 @@ buildscript {
     def roomVersion = System.getenv('CI') ? java7SafeRoomVersion : m1MacSafeRoomVersion
 
     ext.versions = [
-            "kotlin"         : "1.5.32",
+            "kotlin"         : "1.7.0",
```

Comment:
Bump Kotlin version to fix build error encountered on latest Android Studio build.